packages/chip/src/chip.scss

```diff
@@ -0,0 +1,54 @@
+@import 'bootstrap-italia/src/scss/base/functions';
+@import 'bootstrap-italia/src/scss/base/variables';
+@import 'bootstrap-italia/src/scss/base/mixins';
+@import 'bootstrap-italia/src/scss/base/maps';
+@import 'bootstrap-italia/src/scss/base/helpers/visually-hidden';
+@import 'bootstrap-italia/src/scss/components/avatar';
+@import 'bootstrap-italia/src/scss/components/chips';
+
+::slotted(it-icon) {
+  display: inline-flex;
+}
+
+:host {
+  img {
+    width: 100%;
+    height: 100%;
+    border-radius: var(--bs-avatar-radius);
+    object-fit: cover;
+    object-position: center;
+  }
+}
+
+:host([is-disabled]) ::slotted(it-icon) {
+  --bs-icon-primary: var(--bs-icon-disabled);
+  --bs-icon-secondary: var(--bs-icon-disabled);
+  --bs-icon-success: var(--bs-icon-disabled);
+  --bs-icon-danger: var(--bs-icon-disabled);
+  --bs-icon-warning: var(--bs-icon-disabled);
+  --bs-icon-default: var(--bs-icon-disabled);
+}
+
+:host ::slotted(it-icon) {
+  --bs-icon-default: var(--bs-chip-icon-color);
+}
+
+:host([is-disabled]) ::slotted(it-button) {
+  cursor: not-allowed !important;
+}
+
+// Quick fix, bsi styles have wonky vertical alignment
+.chip-label {
+  margin-top: 2px;
```

Comment:
Questa property crea un bug in realtà, il testo è disallineato rispetto ad avatar o icona
<img width="371" height="65" alt="Screenshot 2025-09-19 alle 11 24 29" src="https://github.com/user-attachments/assets/ad576a2c-f8e3-415c-baa6-81e3042f1c81" />